aioslimproto/client.py

```diff
@@ -966,6 +977,20 @@ def _process_setd(self, data: bytes) -> None:
             if display_height:
                 self.display_control.height = display_height
 
+            # Trigger an event callback for "PLAYER_DISPLAY_RESOLUTION"
+            resolution = f"{display_width} x {display_height}"
+            self.callback(self, EventType.PLAYER_DISPLAY_RESOLUTION, resolution)
```

Comment:
should the callback maybe be called after setting these variables below ?

Reply:
do you mean after the check for display.width or display.height == 0 ?